### PR TITLE
Unnecessary assignments are removed.

### PR DESCRIPTION
**Pull request by kamaci**

### CODE REVIEW

File: modules/integration-tests/src/main/java/org/apache/fluo/integration/TestUtil.java [hunk 1]
```diff
@@ -27,14 +27,14 @@ private TestUtil() {}
   private static final Bytes ZERO = Bytes.of("0");
 
   public static void increment(TransactionBase tx, Bytes row, Column col, int val) {
-    int prev = 0;
+    int prev;
     String prevStr = tx.get(row, col, ZERO).toString();
     prev = Integer.parseInt(prevStr);
```
Comment (ctubbsii) [inline modules/integration-tests/src/main/java/org/apache/fluo/integration/TestUtil.java:32]:
```suggestion
    String prevStr = tx.get(row, col, ZERO).toString();
    int prev = Integer.parseInt(prevStr);
```

File: modules/integration-tests/src/main/java/org/apache/fluo/integration/TestUtil.java [hunk 2]
```diff
@@ -27,14 +27,14 @@ private TestUtil() {}
...
     tx.set(row, col, Bytes.of(prev + val + ""));
   }
 
   public static void increment(TransactionBase tx, String row, Column col, int val) {
-    int prev = 0;
+    int prev;
     String prevStr = tx.gets(row, col, "0");
     prev = Integer.parseInt(prevStr);
```
Comment (ctubbsii) [inline modules/integration-tests/src/main/java/org/apache/fluo/integration/TestUtil.java:39]:
```suggestion
    String prevStr = tx.gets(row, col, "0");
    int prev = Integer.parseInt(prevStr);
```

**Review by ctubbsii — CHANGES_REQUESTED** [review]
Thanks for the PR @kamaci ; These look okay to me, but I made an additional suggestion.

**Comment by kamaci** [comment]
I've approved the changes.

**Review by ctubbsii — DISMISSED** [review]
The expression, `prev + val + ""` looks ambiguous to me. I can't tell if it is supposed to be `Integer.toString(prev) + Integer.toString(val)` or `Integer.toString(prev + val)`.

This is an existing problem, and not related to your changes, but if you want to fix it, it would be a good improvement.

Your changes look fine to me, though. Thanks!

**Comment by kamaci** [comment]
> The expression, `prev + val + ""` looks ambiguous to me. I can't tell if it is supposed to be `Integer.toString(prev) + Integer.toString(val)` or `Integer.toString(prev + val)`.
> 
> This is an existing problem, and not related to your changes, but if you want to fix it, it would be a good improvement.
> 
> Your changes look fine to me, though. Thanks!

I've used `prevStr` variable.

File: modules/integration-tests/src/main/java/org/apache/fluo/integration/TestUtil.java [hunk 3]
```diff
@@ -27,17 +27,13 @@ private TestUtil() {}
   private static final Bytes ZERO = Bytes.of("0");
 
   public static void increment(TransactionBase tx, Bytes row, Column col, int val) {
-    int prev = 0;
     String prevStr = tx.get(row, col, ZERO).toString();
-    prev = Integer.parseInt(prevStr);
-    tx.set(row, col, Bytes.of(prev + val + ""));
+    tx.set(row, col, Bytes.of(prevStr + val));
```
Comment (ctubbsii) [inline modules/integration-tests/src/main/java/org/apache/fluo/integration/TestUtil.java:31]: Your latest change forces it to be a string concatenation with `val`. I think it was actually supposed to be a sum first, and then converted to a string, as in `Bytes.of(Integer.toString(Integer.parseInt(prevStr) + val))`

Sorry, I feel like I've led you down a rabbit hole. If you just want to undo this last change, I think your previous changes were fine.

Comment (kamaci) [reply modules/integration-tests/src/main/java/org/apache/fluo/integration/TestUtil.java]: Thanks for the feedback! I've updated the PR.

File: modules/integration-tests/src/main/java/org/apache/fluo/integration/TestUtil.java [hunk 4]
```diff
@@ -27,17 +27,13 @@ private TestUtil() {}
...
-    tx.set(row, col, Bytes.of(prev + val + ""));
+    tx.set(row, col, Bytes.of(Integer.toString(Integer.parseInt(prevStr) + val)));
   }
 
   public static void increment(TransactionBase tx, String row, Column col, int val) {
-    int prev = 0;
     String prevStr = tx.gets(row, col, "0");
-    prev = Integer.parseInt(prevStr);
-    tx.set(row, col, prev + val + "");
+    tx.set(row, col, String.valueOf(Integer.parseInt(prevStr) + val));
```
Comment (ctubbsii) [inline modules/integration-tests/src/main/java/org/apache/fluo/integration/TestUtil.java:36]: I double checked in `jshell` that `int + int + ""` does do the sum first, then convert to a String.

Also, it looks like `String.valueOf(int)` just calls `Integer.toString(int)`, so we can just use that for both.

```suggestion
    tx.set(row, col, Integer.toString(Integer.parseInt(prevStr) + val));
```

**Comment by ctubbsii** [comment]
Thanks for the PR, @kamaci !

**Pull request merged**